pkg/validation/validation.go

```diff
@@ -64,16 +64,24 @@ func NodeGroups(nodeGroups []nropv1.NodeGroup, platf platform.Platform) error {
 		}
 	}
 
```

Comment:
missing `}` for the previous function

Reply:
the old fucntion is not cleaned up in this commit:
https://github.com/openshift-kni/numaresources-operator/pull/1154/commits/a8d4c7acaeb4578f6b0367e6e9fd9d14f288e3a0#diff-feefa601b9dc749f80c19b5769691942730947a8a8226c6ff15767916e58980cR59